### ISSUE TITLE
Unable to load Marketbuddy

### ISSUE THREAD

**Issue by shiki2004** [chalkos/Marketbuddy#19]
### What's the issue?

I wasn't able to load Marketbuddy after install, it shows an Error message and thats the only plugin I install
Restart the game etc nothing fix

![image](https://user-images.githubusercontent.com/81957526/169458017-54041284-2863-4464-8e2f-8cd9bd572f5d.png)

### FFXIV Update

- [X] I have confirmed that I have the latest version of XIVLauncher and Marketbuddy.

**Comment by xPumaa** [comment]
Please send the log that appears during the installation first. You can get it with the command `/xllog`

You can also try deleting the `Marketbuddy` folder in `%appdata%\XIVLauncher\installedPlugins`. If you haven't used Marketbuddy yet, please also delete the `Marketbuddy.json` file in `%appdata%\XIVLauncher\pluginConfigs` and then try to install it again.

**Comment by xPumaa** [comment]
You have forgotten to change the version number.
[Here](https://github.com/chalkos/Marketbuddy/blob/93bb7611a5ca644ff7d4f27749127c06c5feff69/Marketbuddy/Marketbuddy.csproj#L7) and [here](https://github.com/chalkos/Marketbuddy/blob/93bb7611a5ca644ff7d4f27749127c06c5feff69/repo.json#L7)

**Comment by chalkos** [comment]
> You have forgotten to change the version number. [Here](https://github.com/chalkos/Marketbuddy/blob/93bb7611a5ca644ff7d4f27749127c06c5feff69/Marketbuddy/Marketbuddy.csproj#L7) and [here](https://github.com/chalkos/Marketbuddy/blob/93bb7611a5ca644ff7d4f27749127c06c5feff69/repo.json#L7)

yeah i noticed it, i fixed it, tested it, and forgot to push it. should be available now
